test/unit/cloudantBaseService.test.js

```diff
@@ -166,4 +167,102 @@ describe('Test CloudantBaseService', () => {
     assert.ok(service.baseOptions.jar === undefined);
     assert.strictEqual(service.getAuthenticator().tokenManager.url, iamUrl);
   });
+
+  describe('Check keepAlive', () => {
+    const auth = new IamAuthenticator({
+      apikey: '1234',
+      clientId: '345',
+      clientSecret: '234',
+    });
```

Comment:
Why not just use a `NoAuthAuthenticator`?

Reply:
a29c016